validate-specs.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
best to use `env` for portability and force exit on first error as in:

    #!/usr/bin/env bash
    set -e

Reply:
fixed in https://github.com/orchestracities/sdk/pull/1/commits/9785d793a6da33549815722ba4ec0437b592048c